Project.toml

```diff
@@ -1,13 +1,15 @@
 name = "ChainRulesCore"
 uuid = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
-version = "0.4.0"
+version = "0.4.1"
+
+[deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
```

Comment:
Isn't `LinearAlgebra` part of stdlib? Can we not just do `using LinearAlgebra`?

Reply:
Standard libraries still need to go in `[deps]`